packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -197,19 +198,15 @@ function Preview({
   }
 
   const sendInspect = throttle(sendInspectUnthrottled, 50);
-  function resetInspector() {
-    setInspectFrame(null);
-    setInspectStackData(null);
-  }
 
   const shouldPreventInputEvents =
     debugPaused || projectStatus === "refreshing" || !showDevicePreview || !!replayData;
 
-  const shouldPreventFromSendingTouch = isInspecting || !!inspectFrame;
+  const shouldPreventFromSendingTouch = inspectorState === "inspecting" || !!inspectFrame;
 
   function onMouseMove(e: MouseEvent<HTMLDivElement>) {
     e.preventDefault();
-    if (isInspecting) {
+    if (inspectorState === "inspecting") {
```

Comment:
I'd rather have `const isInspecting = inspectorState === "inspecting"` instead of repeating this check in multiple places.